scripts/development/m3_stack/prometheus02.yml

```diff
@@ -1,8 +1,8 @@
 global:
-  external_labels:
-    role: "remote"
-  scrape_interval:     15s
-  evaluation_interval: 15s
+  # external_labels:
+  #   role: "remote_replica2"
```

Comment:
Why did we remove these btw?

Reply:
This is so that both replicas send the same data to the same timeseries - I'll restore it to just be a constant label perhaps.

Reply:
I don't think this tag is explicitly written, it's just "added" to remote read metrics (may be wrong though)

Reply:
Hm the docs arent super helpful

Reply:
Nah, they do get written as well